lib/immagine/image_processor_driver.rb

```diff
@@ -44,10 +43,11 @@ def process
 
       # CONVERT
       image_processor.convert_format!(conversion_format) if conversion_format
-
-      img         = image_processor.img
-      img_quality = quality
-
+      
+      # QUALITY
+      img_quality = Integer(format_processor.quality || 85)
```

Comment:
@ettomatic suggests this could be `format_processor.quality.to_i` instead which I agree would be nicer seeing as the second value is already an integer